certbot-dns-linode/certbot_dns_linode/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""
+The `~certbot_dns_linode.dns_linode` plugin automates the process of
+completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
+subsequently removing, TXT records using the Linode API.
+
+
+Named Arguments
+---------------
+
+==========================================  ===================================
+``--dns-linode-credentials``          Linode credentials_ INI file.
+                                            (Required)
+``--dns-linode-propagation-seconds``  The number of seconds to wait for
+                                            DNS to propagate before asking the
+                                            ACME server to verify the DNS
+                                            record.
+                                            (Default: 930)
```

Comment:
This is a very long time compared to the other DNS plugins (which mostly take 10-60 seconds). It seems unlikely to confuse an interactive user (as we print `Waiting 930 seconds for DNS changes to propagate` right before we wait). However, I wonder what impact this might have on someone with many distinct certificates which need renewal (it seems like we combine the waiting process for each certificate, but do not combine the process _across_ certificates).

I'm not sure there's any change needed as a part of this PR, but wanted to mention it for completeness. We could perhaps call this out more explicitly in the documentation, although it may already be clear enough.